internal/command/project.go

```diff
@@ -187,6 +199,29 @@ func (c *Commands) checkProjectExists(ctx context.Context, projectID, resourceOw
 	return agg.ResourceOwner, nil
 }
 
+func (c *Commands) addProjectMemberEvents(ctx context.Context, resourceOwner, projectID string, admins []*AddProjectAdmin) ([]eventstore.Command, error) {
+	events := make([]eventstore.Command, 0)
```

Comment:
Same condition for length of the list I would do here, does not make any difference for the result, but you avoid allocating a variable which is unnecessary.